src/adagrams.js

```diff
@@ -1,15 +1,111 @@
-export const drawLetters = () => {
+import _ from 'underscore';
```

Comment:
Love to see you researching and applying utility libraries! Underscore is cool, [Lodash](https://lodash.com/) is a very similar library and might be of interest to you.